tools/mini-yetus.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/sh
+# Copyright (c) 2017-2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Do you hate yetus like everyone else? well I can't do anything about that but
+# using this you can at least hate `make yetus` a little less. This script will
+# run yetus on the changes in your current branch compared to master (excluding
+# vendor changes) and output the results, much faster than running it on the
+# entire codebase with `make yetus`.
+
+FULL=false
+
+usage() {
+    echo "Usage: $0 [-b branch_name] [-f]"
```

Comment:
`-b` was replaced with `-s` and `-d` as far as I see

Reply:
fixed.